developer-docs-site/docs/tutorials/run-a-fullnode.md

```diff
@@ -313,6 +313,25 @@ directly there).
 8. See the [Verify initial synchronization](#verify-initial-synchronization) section above for checking if the node
 is syncing again.
 
+## Node-Op FAQ
+
+This section will be updated over time. Feel free to add any Question/Answer pair.
+
+### Question(s):
+When starting the node, it throws a yaml-parsing error
+###  Answer:
+yaml files are very sensitive to formatting errors. Use a dedicated yaml editor and check if the each line is on the correct level
+
+---
+
+###  Question(s):
+I have no peers on the available node-testers  
+I have no output when curl'ing 127.0.0.1:9101/metrics 2> /dev/null | grep "aptos_state_sync_version{type="synced"}"  
+On to available node-testers, it shows that i am not syncing 
+###  Answer:
+Check your public_full_node.yaml if you have added some working peers. To check if peers are working, you can enter the peer-nodes IP address in any of the node-testers and check their status.
```

Comment:
nit: can you change "peers" to "seed peers"? Just to make it obvious to readers that we're talking about seeds, here.
nit: change "node-testers" to "node testers".